### ISSUE TITLE
Footlights

### ISSUE THREAD

**Issue by Jbilotti218** [bethlakshmi/gbe-divio-djangocms-python2.7#481]
We have a request/suggestion for footlights.

Footlights typically are across the front of stage at floor level, aimed upwards to light the lower third of a performer from below.

Stage is flat, with no well or recess for lighting. Fixtures would need to be physically on stage at front edge to be effective.

Concerns to be addressed:
  -Must be DMX controlled to support lighting looks, and blackout needs.
   -Fixture selection is important, not only for stage esthetics, but for color temp and overall look. We do not want the dreaded BLUE SMURF look on all videos/pictures.
   -Stage entrance/exit (Off front of stage) may be limited by equipment.
   -May need box or board to create a stage edge/frame and not look like the backside of lighting fixtures.

**Comment by burlexpo** [comment]
I don't believe Dr. Fluffy is enrolled as a collaborator yet.  When he is, we should assign him this.
I may have some analog footlights (floods in boxes) that have a finished look but would only be one color and require a DMX controlled dimmer pack
N.B. @Jbilotti218 the "Enhancement" tag is specifically for the website.  Technically, almost everything we list here is an enhancement ;-)
